source/draw/Image.ooc

```diff
@@ -26,6 +26,11 @@ CoordinateSystem: enum {
 	YUpward = 0x02
 }
 
+TransformMethod: enum {
+	Fast,
+	Smooth
+}
```

Comment:
Do we need anything more here ? I thought it's better to have it like that than
```ooc
TransformMethod: enum {
    NearestNeighbour,
    Bilinear
}
```
As a user of this class, I wouldn't really care about what method is used under the hood. I would simply want to have a `fast` or `smooth` method of rescaling.
What do you think ?